tests/management/group/test_view.py

```diff
@@ -2129,6 +2131,101 @@ def test_get_group_service_account_invalid_limit_offset(self, mock_request):
             self.assertEqual(int(response.data.get("meta").get("count")), 3)
             self.assertEqual(len(response.data.get("data")), 3)
 
+    def test_get_group_principals_check_service_account_ids_incompatible_query_parameters(self):
```

Comment:
I like all these tests and it would be nice to have some positive test too here

Reply:
I have added a couple of tests for the view:

- A test for mixed results.
- A test for when a different group which does not have any service accounts is specified.

Reply:
thank you ❤️